.changeset/curvy-trains-whisper.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I'd love to know more about this file

Reply:
https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md#i-am-in-a-multi-package-repository-a-mono-repo